### DISCUSSION TITLE
o1-mini 是否可用？昨天貌似可以

### DISCUSSION THREAD

**Discussion by leezee7** [BurnHair/docs#200] [技术支持]
现在报错：

> 当前分组 vip 下对于模型 o1-mini 无可用渠道 (request id: 20240921080853562668560F4xEJ5WT)

**Comment by BennyThink** [comment]
因为我在做测试，开了一小会😂

**Reply by cabudon** [reply]
家人们，上链接，是怕免费额度糟蹋吗？可以设置仅付费额度可用

还有chatgpt-4o-latest，馋死了

**Reply by zakizou** [reply]
> 因为我在做测试，开了一小会😂

o1-mini最近会上吗？

**Reply by BennyThink** [reply]
会，请等消息～

**Comment by BennyThink** [comment]
可用了
